carbon_black_cloud/plugin.spec.yaml

```
@@ -0,0 +1,456 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: carbon_black_cloud
+title: Carbon Black Cloud
+description: The VMware Carbon Black Cloud is a cloud-native endpoint protection platform (EPP) that combines the intelligent system hardening and behavioral prevention needed to keep emerging threats at bay, using a single lightweight agent and an easy-to-use console
```

Shorten this description so it displays better on the Ext Lib search page. We'll keep the long description for help.md

```suggestion
description: The VMware Carbon Black Cloud is a cloud-native endpoint protection platform (EPP).
```